configure.ac

```diff
@@ -160,6 +160,7 @@ AS_IF(
  [test "x$enable_offline" != "xfalse"],
  [[CFLAGS="$CFLAGS -DACVP_OFFLINE"]
   [LDFLAGS="-static $LDFLAGS"]
+  [CC="$fomdir/bin/fipsld"]
```

Comment:
Might want to add $CC in there in case the user passed in more externally

Reply:
Normally yes, but in the case of using fipsld for compilation there can be no other options included.